src/services/notifyService.ts

```diff
@@ -0,0 +1,102 @@
+import { Prisma } from '@prisma/client';
+
+interface Notifier {
+  notifyNewsPost(post: Prisma.NewsPostGetPayload<{}>): void;
+}
+
+export default class NotifyService {
+  static notifyNewsPost(post: Prisma.NewsPostGetPayload<{}>) {
+    notifiers.forEach((notifier) => {
+      notifier.notifyNewsPost(post);
+    });
+  }
+}
+
+class DiscordWebhookNotifier implements Notifier {
+  private webhook: string;
+
+  public constructor(webhook: string) {
+    this.webhook = webhook;
+  }
+
+  notifyNewsPost(post: Prisma.NewsPostGetPayload<{}>) {
+    fetch(this.webhook, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        content: `Nyhet publicerad av **${post.writtenByCid}**`,
+        embeds: [
+          {
+            title: post.titleSv,
+            url: `http://localhost:3000/post/${post.id}`,
```

Comment:
This should not be localhost but maybe we fix this later via a config?

Reply:
I've gone ahead and added an environment variable (`BASE_URL`) to solve this